### PR TITLE
DGC Detail Screen (EXPOSUREAPP-7478)

### PR DESCRIPTION
**Pull request by axelherbstreith**
- [x] Add Alert when deleting test certificate
- [x] Add Fullscreen Qr-Code
- [x] Wire Screen
- [x] Connect to Database

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/details/CovidCertificateDetailsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    }
+
+    private fun FragmentCovidCertificateDetailsBinding.onCertificateReady(
+        testCertificate: TestCertificate
+    ) {
+        qrCodeCard.subtitle.text = getString(
+            R.string.detail_green_certificate_card_subtitle,
+            testCertificate.testResultAt.toShortDayFormat(),
+            testCertificate.testResultAt.toShotTimeFormat(),
+        )
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/details/CovidCertificateDetailsFragment.kt:70]: @d4rken  which date should be used here ?
Screenshot 2021-06-04 at 15 42 12

Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/details/CovidCertificateDetailsFragment.kt:70]: `dr`, `testResultAt` would be correct AFAIK, I saw in another PR though that the `dr` attribute is optional now, so we need to clarify whether this is still desired and if there needs to be a fallback value :thinking: 

/cc @mlenkeit

Comment (mlenkeit) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/details/CovidCertificateDetailsFragment.kt:70]: `sc` should be used here.

Comment (d4rken) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/greencertificate/ui/certificates/details/CovidCertificateDetailsFragment.kt:70]: > `sc` should be used here.

Thanks, so `sampleCollectedAt` @axelherbstreith

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

**Review by BMItr — COMMENTED** [review]
code looks good to me 👌 
just ping when you are done wiring ;-)

**Review by AlexanderAlferov — DISMISSED** [review]
LGTM

**Review by AlexanderAlferov — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**